applications/GeoMechanicsApplication/custom_conditions/U_Pw_normal_lysmer_absorbing_condition.cpp

```diff
@@ -0,0 +1,588 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//
+//  Main authors:    Aron Noordam
+//
+
+#include <math.h>
+// Application includes
+#include "custom_conditions/U_Pw_normal_lysmer_absorbing_condition.hpp"
+
+namespace Kratos
+{
+
+template< unsigned int TDim, unsigned int TNumNodes >
+Condition::Pointer UPwLysmerAbsorbingCondition<TDim,TNumNodes>::Create(IndexType NewId,NodesArrayType const& ThisNodes,PropertiesType::Pointer pProperties) const
+{
+    return Condition::Pointer(new UPwLysmerAbsorbingCondition(NewId, this->GetGeometry().Create(ThisNodes), pProperties));
+}
+
+//----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Calculates LHS stiffness part of absorbing boundary
+/// </summary>
+/// <param name="rLhsMatrix"></param>
+/// <param name="rRightHandSideVector"></param>
+/// <param name="CurrentProcessInfo"></param>
+template< unsigned int TDim, unsigned int TNumNodes >
+void UPwLysmerAbsorbingCondition<TDim, TNumNodes>::CalculateLocalSystem(MatrixType& rLhsMatrix, VectorType& rRightHandSideVector, const ProcessInfo& CurrentProcessInfo)
+{
+    int n_dof = TNumNodes * TDim;
```

Comment:
Use 

static constexpr SizeType N_DOF = TNumNodes * TDim;

and move it to the header file. Then you have access to that everywhere.